django/contrib/admin/checks.py

```diff
@@ -70,10 +87,8 @@ def check_dependencies(**kwargs):
             id='admin.E403',
         ))
     else:
-        if ('django.contrib.auth.context_processors.auth'
-                not in django_templates_instance.context_processors and
-                'django.contrib.auth.backends.ModelBackend'
-                in settings.AUTHENTICATION_BACKENDS):
+        if ('django.contrib.auth.context_processors.auth' not in django_templates_instance.context_processors and
+                _contains_subclass('django.contrib.auth.backends.ModelBackend', settings.AUTHENTICATION_BACKENDS)):
```

Comment:
This line is horrible. 😄 

Simon mentioned this in his review before but, maybe leaving up until the `and\n` as is and putting the new check on the next line, to keep the diff that bit smaller...? (I don't mind this as it is per se — whichever way we do it, it is long and horrible.)

Reply:
Haha, agreed. But good point, I've made adjustments keeping the diff to a minimum now.